### PR TITLE
vmm: add vcpus of clh to sandbox cgroup

### PR DESCRIPTION
**Pull request by Burning1020**
Add vcpu thread of cloud hypervisor process to sandbox vcpu cgroup.
This is not include virtiofsd child process if have.

### CODE REVIEW

File: vmm/sandbox/src/stratovirt/mod.rs [hunk 1]
```diff
@@ -164,7 +166,7 @@ impl VM for StratoVirtVM {
 
         if let Err(e) = self.wait_stop(Duration::from_secs(10)).await {
             if force {
-                if let Ok(pid) = self.pid().await {
+                if let Some(pid) = self.pids.vmm_pid {
```
Comment (flyflypeng) [inline vmm/sandbox/src/stratovirt/mod.rs:169]: Like cloud_hypervisor style, add condition with `pid==0` ?

File: vmm/sandbox/src/cloud_hypervisor/mod.rs [hunk 2]
```diff
@@ -114,7 +116,7 @@ impl CloudHypervisorVM {
         ))
     }
 
-    async fn start_virtiofsd(&self) -> Result<()> {
+    async fn start_virtiofsd(&self) -> Result<Option<u32>> {
```
Comment (abel-von) [inline vmm/sandbox/src/cloud_hypervisor/mod.rs:119]: 这个函数为啥要返回Result<Option<>> 返回一个Result就行吧？

File: vmm/sandbox/src/stratovirt/mod.rs (hunk 1 above)
Comment (Burning1020) [reply vmm/sandbox/src/stratovirt/mod.rs]: Updated

File: vmm/sandbox/src/cloud_hypervisor/mod.rs (hunk 2 above)
Comment (Burning1020) [reply vmm/sandbox/src/cloud_hypervisor/mod.rs]: Updated

**Review by flyflypeng — APPROVED** [review]

**Pull request merged**